jib-core/src/main/java/com/google/cloud/tools/jib/api/JibContainerBuilder.java

```diff
@@ -488,6 +489,31 @@ public JibContainerBuilder setCreationTime(Instant creationTime) {
     return this;
   }
 
+  /**
+   * Adds a platform to the platforms configurations for the container.
```

Comment:
Let's copy the Javadoc from `ContainerBuildPlan`.

```suggestion
   * Adds a desired image platform (OS and architecture pair). If the base image reference is a
   * Docker manifest list or an OCI image index, an image builder may select the base image
   * matching the given platform. If the base image reference is an image manifest, an image
   * builder may ignore the given platform and use the platform of the base image or may decide to
   * raise on error.
   *
   * <p>Note that a new build plan starts with "amd64/linux" as the default platform. If you want
   * to reset the default platform instead of adding a new one, use {@link #setPlatforms(Set)}.
```